project/src/main/webapp/header.html

```diff
@@ -2,10 +2,6 @@
 <html>
   <head>
     <meta charset="UTF-8" />
-    <link
```

Comment:
oh can you put this back? it's for the website font

Reply:
Done.